src/util/simplify_expr.cpp

```diff
@@ -103,24 +103,24 @@ simplify_exprt::resultt<> simplify_exprt::simplify_abs(const abs_exprt &expr)
   return unchanged(expr);
 }
 
-simplify_exprt::resultt<> simplify_exprt::simplify_sign(const exprt &expr)
+simplify_exprt::resultt<> simplify_exprt::simplify_sign(const sign_exprt &expr)
 {
   if(expr.operands().size()!=1)
     return unchanged(expr);
```

Comment:
I don't think this should be necessary anymore.

Reply:
removed